README.md

```diff
@@ -1920,19 +1927,23 @@ We’ll discuss negotiation strategies later on in this chapter, but keep in min
 everything we present here could come up in your negotiations.”
 
 Companies will always ask you what you want for compensation, and you should always be
-cautious about answering.
+[cautious about answering](http://review.chicagobooth.edu/strategy/2018/article/how-answer-one-toughest-interview-questions).
```

Comment:
So asking for salary expectations is different than asking for salary history. Since banning current and historical salary questions is related and relevant, I added a bullet about it. Used my first 🔸 ! It will be in my next commit.

Reply:
💪 🎉